wooting-analog-common/src/lib.rs

```diff
@@ -576,4 +455,4 @@ pub enum HIDCodes {
     RightShift = 0xe5, //SHIFT_RIGHT
     RightAlt = 0xe6,   //ALT_RIGHT
     RightMeta = 0xe7,  //META_RIGHT
-}
+}
```

Comment:
Please add back the new lines to the end of the files you modified

Reply:
oops, didnt even notice